### PR TITLE
pylint all py files

### PR DESCRIPTION
**Pull request by krzyzacy**
borrow the remote bazel

### CODE REVIEW

**Comment by krzyzacy** [comment]
hummmm where are modules like `external.pylint.test.extensions.test_check_docs` comes from

**Comment by spxtr** [comment]
`external/` might be created by bazel, you should skip it.

**Comment by krzyzacy** [comment]
@spxtr @ixdy does `--prune` not work in bazel? The `find` command works differently than on my local machine...

File: verify/verify-pylint.sh [hunk 1]
```diff
@@ -20,11 +20,6 @@ set -o pipefail
...
 
-# TODO(fejta): all python files
-pylint scenarios/*.py
-pylint jenkins/bootstrap.py
-pylint jenkins/janitor.py
-pylint jobs/*.py
-pylint queue-health/graph/graph.py
-pylint queue-health/weekly_commit_stats.py
-pylint triage/*.py
+find . -type d \( -path gubernator/third_party -o -path vendor -o -path external \) -prune -o -name *.py
```
Comment (ixdy) [inline verify/verify-pylint.sh]: I think you want
`find . -type d \( -path ./gubernator/third_party/ -o -path ./vendor/ -o -path ./external/ \) -prune -o -name *.py`

Comment (krzyzacy) [inline verify/verify-pylint.sh]: I tried both.. I'll try `-not` instead

File: verify/verify-pylint.sh [hunk 2]
```diff
@@ -20,11 +20,4 @@ set -o pipefail
...
 
-# TODO(fejta): all python files
-pylint scenarios/*.py
-pylint jenkins/bootstrap.py
-pylint jenkins/janitor.py
-pylint jobs/*.py
-pylint queue-health/graph/graph.py
-pylint queue-health/weekly_commit_stats.py
-pylint triage/*.py
+find . -type f -name *.py -not -path "./external*" -not -path "./gubernator/third_party*" -print | xargs pylint
```
Comment (ixdy) [inline verify/verify-pylint.sh]: the bazel runfiles tree is all symlinks. to use `-type f` you need to do `find -L . -type f ...`

Comment (krzyzacy) [inline verify/verify-pylint.sh]: ohhhh that's why :-p thank for pointing!

**Pull request closed without merging**